client/landing/stepper/declarative-flow/link-in-bio-domain.ts

```diff
@@ -54,8 +54,8 @@ const linkInBioDomain: Flow = {
 				asyncComponent: () => import( './internals/steps-repository/design-carousel' ),
 			},
 			{
-				slug: 'siteCreationStep',
-				asyncComponent: () => import( './internals/steps-repository/site-creation-step' ),
+				slug: 'createSiteStep',
```

Comment:
This should be `createSite` so we get the correct mapping.
```suggestion
				slug: 'createSite',
```